internal/integration/telemetry_sdk_emitter.go

```diff
@@ -238,15 +238,17 @@ func (te *TelemetryEmitter) emitSummary(metric Metric, timestamp time.Time) erro
 		return fmt.Errorf("unknown summary metric type for %q: %T", metric.name, metric.value)
 	}
 
-	te.harvester.RecordMetric(telemetry.Summary{
-		Name:       metric.name + "_sum",
-		Attributes: metric.attributes,
-		Count:      1,
-		Sum:        summary.GetSampleSum(),
-		Min:        math.NaN(),
-		Max:        math.NaN(),
-		Timestamp:  timestamp,
-	})
+	if sumCount, ok := te.deltaCalculator.CountMetric(metric.name+"_sum", metric.attributes, float64(summary.GetSampleSum()), timestamp); ok {
+		te.harvester.RecordMetric(telemetry.Summary{
+			Name:       metric.name + "_sum",
+			Attributes: metric.attributes,
+			Count:      1,
+			Sum:        sumCount.Value,
```

Comment:
I would thought that this value already come in the original prometheus metric and we just need it to send it as it is.

Reply:
In prometheus it comes as a counter and so in order for us to be able to calculate the average (_sum/_count) we also need to send it as a delta, like we do for the _count